src/Data/LLVM/BitCode/IR/Globals.hs

```diff
@@ -50,18 +50,37 @@ parseGlobalVar n r = label "GLOBALVAR" $ do
   vis <- if length (recordFields r) > (6 + offset) && not (link `elem` [Internal, Private])
                 then field 6 visibility
                 else pure DefaultVisibility
+  tl <- if length (recordFields r) > (7 + offset) -- && not (link `elem` [Internal, Private])
```

Comment:
Can this comment be deleted? I think thread locality is parsed independently of the linkage, based on my reading of the LLVM implementation [here](https://github.com/llvm/llvm-project/blob/803bd812b16d1454b8fc0d6d66d7da793408b34d/llvm/lib/Bitcode/Reader/BitcodeReader.cpp#L4086-L4088).